### DISCUSSION TITLE
New placeholders

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#228] [Ideas]
Party state (this is used to make events with the conditionalevents plugin)
%Eden_party_status%
You could give a variable that is type In one/None

player deaths

%Eden_total_deaths%
%Eden_unranked_deaths%
etc...

Status of an arena/kit

%Eden_arena_status%
%Eden_kit_status%
%Eden_arena_name%

Party status (If it is public or private)

%Eden_party_opening%

I hope and my suggestions are taken into account, thank you very much.

**Comment by liexes** [comment]
@GoodestEnglish please add this

**Comment by GoodestEnglish** [comment]
The only placeholder I could add is kit status and party status. Here's why:
Eden doesn't track death count. Eden tracks loss count. You can use `eden_player_overall_win` and `eden_player_overall_loss` for the placeholders.
An arena can represent multiple, we call it ArenaDetail in Eden. Which basically means an arena can be multiple actual arena (For example, you have an arena called `sumo01`, but in Eden, this arena can have multiple actual arena because the arena duplicate system in Eden)

**Reply by GoodestEnglish** [reply]
The two placeholders I have said will be added in next version.

It will be called `%eden_kit_status_<kit name>%` and `%eden_party_privacy%`

**Reply by liexes** [reply]
Ok, but please add %eden_in_party% to give if the player is in party o no, with this i can create multiple things with conditionalevents plugin

**Reply by GoodestEnglish** [reply]
Alright

**Reply by liexes** [reply]
It would also serve a placeholder to return if the player is or in game and in which it %eden_kit_infight% this will throw if it is not in game will throw a "no" and if it is in game will throw the name of the kit, this adds a gln wide capacity of possibilities to do things with the Plugin conditional events.

**Reply by liexes** [reply]
sorry to bother you so much
